releases/net471/KnownIssues/481090-WPF Touch generates NullReferenceException in ProcessInputReport.md

```diff
@@ -0,0 +1,12 @@
+## WPF Touch generates a NullReferenceException in WispLogic.ProcessInputReport
+
+### Symptoms
+A NullReferenceException is thrown and the application might crash.
+
+### Cause
```

Comment:
Is this an app compat issue or a bug. @PriyaPurkayastha, what do you think?

Reply:
Ron, I think this PR is for .NET 4.7.1 Known Issues documentation, not for .NET 4.7.1 compat documentation